docs/getting-started/05-add-redis-service.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Add the Redis service
+type: Getting Started
+---
+
+Provision the external [Redis](https://redis.io/) service that will replace your microservice in-memory storage. To do so, you will use a sample addon configuration linking to an example in the GitHub repository.
+
+## Reference
+
+This guide demonstrates how [Service Catalog](/components/service-catalog/) works in Kyma. It enables easy access to services ([ServiceClass CRs](https://svc-cat.io/docs/resources/#service-classes)) managed by [Service Brokers](/components/service-catalog/#overview-service-brokers) registered in Kyma. You can consume these services by creating their instances and binding them to your microservices and Functions.
```

Comment:
```suggestion
This guide demonstrates how [Service Catalog](/components/service-catalog/) works in Kyma. It enables easy access to services ([ServiceClass custom resources](https://svc-cat.io/docs/resources/#service-classes)) managed by [Service Brokers](/components/service-catalog/#overview-service-brokers) registered in Kyma. You can consume these services by creating their instances and binding them to your microservices and Functions.
```
I didn't unfold this acronym here as there are already parentheses in this place. I added a comment to line 22 instead.
If you'd rather reword it to have it here, feel free to do so.